examples/ex37.hpp

```diff
@@ -422,12 +513,8 @@ void DiffusionSolver::SetupFEM()
    }
 }
 
-void DiffusionSolver::Solve()
+void DiffusionSolver::AssembleBoundary()
```

Comment:
Why is this a separate function now?

Reply:
I suggest combining the BC function into the bilinear form assembly function with `update_BCs` as an optional boolean argument. This is safer for the end user who could accidentally call these functions in the wrong order.